src/main/java/com/kodilla/ecommercee/entity/UserOrder.java

```diff
@@ -19,4 +21,17 @@
     @Id
     @GeneratedValue
     private Long Id;
+
+    private Date orderMade;
```

Comment:
Proponuje używać LocalDate/LocalDateTime zamiast Date. To nowsze API które powinno zastępować używanie Date.
Jeśli chcesz mieć w przyszłości date ze strefa czasową (bo klasa Date tak przechowuje dane) to możesz zamiast LocalDateTime uzyć ZonedDateTime